src/node_sqlite.cc

```diff
@@ -930,7 +930,8 @@ bool StatementSync::BindParams(const FunctionCallbackInfo<Value>& args) {
   int anon_idx = 1;
   int anon_start = 0;
 
-  if (args[0]->IsObject() && !args[0]->IsUint8Array()) {
+  if (args[0]->IsObject() && !args[0]->IsArrayBufferView() &&
+      !args[0]->IsDataView()) {
```

Comment:
`ArrayBuffer.isView(new DataView(new ArrayBuffer))` is `true`, I think that third check is pure overhead and already covered by the second one
```suggestion
  if (args[0]->IsObject() && !args[0]->IsArrayBufferView()) {
```